web/package.json

```diff
@@ -33,8 +34,10 @@
     "crypto-js": "4.1.1",
     "d3-dag": "0.9.1",
     "date-fns": "2.28.0",
+    "html-react-parser": "3.0.4",
     "jmespath": "0.16.0",
     "lodash": "4.17.21",
+    "markdown-it": "13.0.1",
```

Comment:
Do we need both dependencies?

Reply:
I think so, one formats the markdown to HTML. and the other one renders the HTML